app/bundles/NotificationBundle/Entity/Notification.php

```diff
@@ -156,6 +161,9 @@ public static function loadMetadata(ORM\ClassMetadata $metadata)
         $builder->createField('message', 'text')
             ->build();
 
+        $builder->createField('button', 'text')
```

Comment:
This will require a migration. `php app/console mautic:migrations:generate` will give you a skeleton class in app/migrations and check the existing migrations for examples.

If you do `php app/console doctrine:schema:update --dump-sql` it'll give you the queries (you'll need to delete the tables you have now for that to work). It's important that we have a schema that matches what Doctrine generates so if you manually created the table schema, it'll need to be redone by Doctrine anyway.